eZ/Bundle/EzPublishRestBundle/Resources/config/services.yml

```diff
@@ -226,6 +226,15 @@ services:
         arguments:
             - "@ezpublish.api.service.search"
 
+    ezpublish_rest.controller.session:
+        class: eZ\Publish\Core\REST\Server\Controller\SessionController
+        parent: ezpublish_rest.controller.base
+        arguments:
+            - '@ezpublish_rest.session_authenticator'
+            - '%ezpublish_rest.csrf_token_intention%'
+            - '@?security.csrf.token_manager'
+            - '@?security.csrf.token_storage'
```

Comment:
Did we ? Maybe...

Gotta admit that I actually prefer single quotes because:
- they don't require that you escape `\`
- they're demanded by our coding standards for PHP (but not for YAML, I agree)

But it is indeed not consistent with what we've done so far.